cmd/bblfshd/main.go

```diff
@@ -66,6 +69,7 @@ func init() {
 	log.level = cmd.String("log-level", "info", "log level: panic, fatal, error, warning, info, debug.")
 	log.format = cmd.String("log-format", "text", "format of the logs: text or json.")
 	log.fields = cmd.String("log-fields", "", "extra fields to add to every log line in json format.")
+	runPProf = cmd.Bool("pprof", false, "run pprof http endpoint.")
```

Comment:
I remember already adding `pprof` support for bblfshd and indeed https://github.com/bblfsh/bblfshd/commit/b1b2883dbdc712570d16ce3edf1345162a6acf2f - it seems to be gone for some reason.

@dennwc do you think we could restore descriptive CLI flags in here please? Exposing `profiler` and `profiler-address`?

Reply:
Done!